en/ros/mavros_installation.md

```diff
@@ -32,7 +32,7 @@ Kinetic also supports Debian Jessie amd64 and arm64 (ARMv8).
 Use `apt-get` for installation:
```

Comment:
I tried to run the instructions below, and they assume that you have `$ROS_DISTRO` set. That might be true, but wasn't when I tried. 

So how about this additional change:
```suggestion
Use `apt-get` for installation, where `${ROS_DISTRO}` below is either `kinetic` or `noetic`:
```

@Jaeyoung-Lim You OK with this (and to merge if it goes in)?

Reply:
Hmm this is a ros environment variable, so should be there if you have ros. May I ask you your setup?

Reply:
@hamishwillee on my system:
```
$ echo $ROS_DISTRO
melodic
```

Reply:
This was foxy. But I set it up a while ago and did not retest for this.
I have edited my suggestion above to change the implication from "you need to modify this command to match your distribution" to "this will set the correct value for your distribution to one of these things". People shouldn't need to know this, but it will do no harm.

merging!